addons/web_tour/static/src/js/running_tour_action_helper.js

```diff
@@ -148,12 +148,43 @@ var RunningTourActionHelper = core.Class.extend({
             $to = get_jquery_element_from_selector(to);
         }
 
-        let toCenter = calculateCenter();
+        let toCenter = this._calculateCenter($to, to);
         values.$element.trigger($.Event("mousemove", {which: 1, pageX: toCenter.left, pageY: toCenter.top}));
         // Recalculate the center as the mousemove might have made the element bigger.
-        toCenter = calculateCenter();
+        toCenter = this._calculateCenter($to, to);
         values.$element.trigger($.Event("mouseup", {which: 1, pageX: toCenter.left, pageY: toCenter.top}));
-     },
+    },
+    _drag_move_and_drop: function (values, to) {
+        let initialMoveOffset = [1, 0];
+        let fromSelector;
+        const elementCenter = values.$element.offset();
+        elementCenter.left += values.$element.outerWidth() / 2;
+        elementCenter.top += values.$element.outerHeight() / 2;
+        // Check if initial move offset is defined: '[deltaX,deltaY]@from => actualTo'.
+        const parts = /^\[(.+),(.+)\]@(.+) => (.+)/.exec(to);            
+        if (parts) {
```

Comment:
That `if` is not useful anymore, right? Now that it got moved in its own method, we can just assume the action will be used with correct parameters?